vgen/src/test/java/io/gapi/vgen/testdata/go_library.baseline

```diff
@@ -75,6 +76,22 @@ var (
 	bookPathTemplate = gax.MustCompilePathTemplate("shelves/{shelf}/books/{book}")
 )
 
+func idempotentRetryCodes() []gax.CallOption {
+	return []gax.CallOption{
+		gax.WithRetryCodes([]codes.Code{
+			codes.DeadlineExceeded,
+			codes.Unavailable,
+		}),
```

Comment:
Is it `ResourceExhausted`?

I saw its description: https://godoc.org/google.golang.org/grpc/codes#Code

I'm not sure if it applies, maybe @tbetbetbe can weigh in?